src/sentry/rules/processing/delayed_processing.py

```diff
@@ -0,0 +1,32 @@
+import logging
+from collections.abc import Mapping
+
+from sentry.buffer.redis import RedisBuffer
+from sentry.models.project import Project
+from sentry.utils.safe import safe_execute
+
+logger = logging.getLogger("sentry.rules.delayed_processing")
+
+
+# TODO: Import once available
+PROJECT_ID_BUFFER_LIST_KEY = "project_id_buffer_list"
```

Comment:
This actually might be a good place to just have it defined, right now in https://github.com/getsentry/sentry/blob/master/tests/sentry/buffer/test_redis.py#L271 I have it defined just because it wasn't anywhere else yet, so if you want you could import it from here in the test.